config-example.yaml

```diff
@@ -1,18 +1,25 @@
 publicHLSPath: webroot/hls
 privateHLSPath: hls
-ffmpegPath: /usr/local/bin/ffmpeg
+ffmpegPath: /usr/bin/ffmpeg
 webServerPort: 8080
-enableOfflineImage: true
 
 videoSettings:
   chunkLengthInSeconds: 4
   streamingKey: abc123
-  encoderPreset: superfast # https://trac.ffmpeg.org/wiki/Encode/H.264
-  passthrough: true # Enabling this will ignore the below stream qualities and pass through the same quality that you're sending it
-  offlineImage: doc/logo.png # Is displayed when a stream ends
+  offlineContent: static/offline.m4v # Is displayed when a stream ends
```

Comment:
This isn't a great name.  It's not clear that it's the content that is shown when the stream is offline.